tests/baselines/reference/castExpressionParentheses.js

```diff
@@ -76,9 +76,9 @@ a;
 a[0];
 a.b["0"];
 a().x;
-1..foo;
-1..foo;
-1.0.foo;
+(1).foo;
+(1.).foo;

```

Comment:
It might be worth re-using the original style from the input here (something that is the explicit intention of some related lines that I was tweaking) and I can fix this particular line by doing that.